infra/gp-multena/values.yaml

```diff
@@ -0,0 +1,28 @@
+multena:
+  dev: false
+  logLevel: DEBUG
+  adminGroup: Gepardec
+  jwksCertUrl: https://sso.apps.c-gepa-play.gepa.vshnmanaged.net/realms/internal/protocol/openid-connect/certs
+  thanosUrl: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+  lokiUrl: https://loki-gateway-http.openshift-logging.svc.cluster.local:8080
+  promLabelUrl: http://localhost:9095
+  provider: configmap
+  tenantLabel: namespace
+  db:
+    user: multitenant
+    password: SafePassword1234!
+    host: localhost
+    port: 3306
+    dbName: blubb
+    query: "SELECT * FROM users WHERE username = ?"
+image:
+  repository: ghcr.io/gepaplexx/multena-proxy
+  pullPolicy: Always
+  tag: "feature-loki"
+
+serviceAccount:
+  create: true
```

Comment:
weggeben. das bricht in der aktuellen umsetzung nur dinge, bringt aber nichts.

Reply:
Was genau funktioniert nicht, wenn kein Service Account erstellt wird? Benötigt vorwiegend die APA, weil die keinen brauchen.